project/build.properties

```diff
@@ -1 +1,2 @@
-sbt.version=0.13.17
\ No newline at end of file
+sbt.version=0.13.17
+#sbt.version=1.2.7
```

Comment:
What was the reason we're still on 0.13.x ? Do we depend on a legacy sbt plug-in or so? Or was it Java 6 support?